app/org/sagebionetworks/bridge/dynamodb/DynamoUserConsentDao.java

```diff
@@ -87,25 +105,80 @@ public UserConsent getUserConsent(String healthCode, StudyIdentifier studyIdenti
     public long getNumberOfParticipants(StudyIdentifier studyIdentifier) {
         DynamoDBScanExpression scan = new DynamoDBScanExpression();
         scan.setConsistentRead(true);
-
         Condition condition = new Condition();
         condition.withComparisonOperator(ComparisonOperator.EQ);
         condition.withAttributeValueList(new AttributeValue().withS(studyIdentifier.getIdentifier()));
         scan.addFilterCondition("studyKey", condition);
 
-        // We need the count of unique study participants, users may end up signing 
-        // more than one version of the consent.
+        // Must find the complete set of unique health codes in both tables.
         Set<String> healthCodes = Sets.newHashSet();
-        List<DynamoUserConsent2> mappings = mapper.scan(DynamoUserConsent2.class, scan);
-        for (DynamoUserConsent2 consent : mappings) {
-            healthCodes.add(consent.getHealthCode());
-        }
+        List<DynamoUserConsent2> mappings2 = mapperV2.scan(DynamoUserConsent2.class, scan);
+        mappings2.stream().forEach(consent -> healthCodes.add(consent.getHealthCode()));
+        
+        // The name of the column has changed, create a different expression
+        scan = new DynamoDBScanExpression();
+        scan.setConsistentRead(true);
+        condition = new Condition();
+        condition.withComparisonOperator(ComparisonOperator.EQ);
+        condition.withAttributeValueList(new AttributeValue().withS(studyIdentifier.getIdentifier()));
+        scan.addFilterCondition("studyIdentifier", condition);
+        
+        List<DynamoUserConsent3> mappings3 = mapperV3.scan(DynamoUserConsent3.class, scan);
+        mappings3.stream().forEach(consent -> healthCodes.add(consent.getHealthCode()));
```

Comment:
Get number of participants is tested in DynamoUserConsentDaoTest because I wanted to verify the query filers are correct.  I could do a unit test of the DAO but not sure what it would test.